0003-placeholders-in-conditions.md

```diff
@@ -0,0 +1,63 @@
+- Start Date: 2023-06-15
+- Target Major Version: 2.x
+- Reference Issues: https://github.com/cedar-policy/cedar/issues/81
+- Implementation PR:
+
+# Summary
+
+__Support template placeholders in conditions__
+
+This RFC proposes to allow the `?principal` and `?resource` placeholders to appear in the `when`/`unless` conditions of a policy, not just in the policy scope.
```

Comment:
To be clear: The restrictions on the use of `?principal` and `?resource` in policy scope remain. For example, you would not be allowed to write
```
permit(
  principal,
  action,
  ?resource in resource);
```
You'd have to write instead
```
permit(principal, action, resource) 
when { ?resource in resource }
```
I.e., the slot should always be on the RHS of `==` or `in` when in the scope.

Reply:
Added a "Non-proposals" section to this RFC to make this clear.